src/components/Marketplace/FarmCard.js

```diff
@@ -0,0 +1,228 @@
+import React, { useState } from 'react';
+import './FarmCard.css';
+import {
+  Card, CardContent, Chip, Collapse, Grid, IconButton,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
+import CheckCircleOutlineIcon from '@material-ui/icons/CheckCircleOutline';
+import PropTypes from 'prop-types';
+
+/* Custom styling for material-ui components */
+const useStyles = makeStyles({
+  cardContainer: { // Overall card container
+    background: 'white', // change to lightblue for testing
+    borderWidth: '0px',
+    borderColor: '#53AA48',
+    margin: '5%',
+    borderRadius: '20px',
+    fontFamily: 'Work Sans',
+  },
+  collapsedBorder: {
+    borderLeftWidth: '15px',
+  },
+  collapsedContent: {
+    width: '95%',
+    marginRight: '2%',
+    marginLeft: '2%',
+  },
+  expandedBorder: { // Border if card is expanded
+    borderTopWidth: '20px',
+  },
+  expandedContent: {
+    width: '93%',
+    marginRight: '2%',
+    marginLeft: '2%',
+    fontSize: '16px',
+  },
+  headerContainer: { // Header contains title, subtitle, + icon
+    color: '#373737',
+    fontFamily: 'Work Sans',
+    marginTop: '3%',
+    marginLeft: '4%',
+    paddingRight: '8%',
+  },
+  titleContainer: {
+    width: '55%',
+  },
+  titleText: {
+    fontSize: '22px',
+  },
+  subTitleText: {
+    fontSize: '18px',
+    marginTop: '3px',
+  },
+  operationTags: {
+    background: '#DBE2ED', // grey
+    color: '#373737', // black
+    borderRadius: '6px',
+    marginTop: '-3%',
+    marginRight: '2%',
+    paddingLeft: '0.8%',
+    paddingRight: '0.8%',
+    fontFamily: 'Work Sans',
+    fontSize: '18px',
+  },
+  marketTags: {
+    background: '#DBE2ED',
+    color: '#373737',
+    borderRadius: '6px',
+    marginLeft: '1%',
+    marginRight: '0.5%',
+    paddingLeft: '0.7%',
+    paddingRight: '0.7%',
+    fontFamily: 'Work Sans',
+    fontSize: '16px',
+  },
+  capabilityTags: {
+    background: '#FFFFFF',
+    color: '#373737',
+    marginTop: '1%',
+    marginRight: '1.3%',
+    borderRadius: '6px',
+    fontFamily: 'Work Sans',
+    fontSize: '16px',
+  },
+  descriptionContainer: {
+    marginTop: '-1.5%',
+    marginBottom: '2%',
+  },
+  marketContainer: {
+    marginTop: '3%',
+  },
+  iconColour: {
+    color: '#53AA48', // green
+  },
+});
+
+export default function FarmCard(props) {
+  const styles = useStyles();
+  const [isExpanded, setIsExpanded] = useState(false);
+  const {
+    farmName, address, zipCode, description, operationTypeTags, marketTags, isPACA, isColdChain,
+    isDelivery,
+  } = props;
+
+  // Set capabilityTags (PACA, cold chain, delivery)
```

Comment:
We no longer need to showcase the isPACA, isColdChain, and isDelivery parts of this component. You can model this figma picture, but make sure to remove the delivery and PACA checkbox! After a call to F2P, we learned that we want F2P to handle all transportation logistics, because it allows them to be control all transportation logstics and is more conducive of their distribution model!


<img width="634" alt="Screen Shot 2021-02-11 at 1 59 37 AM" src="https://user-images.githubusercontent.com/59102166/107622547-dbebc080-6c0c-11eb-9be3-0c03eeb080f3.png">